package.json

```diff
@@ -8,6 +8,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.15",
+    "@types/react-virtualized": "^9.21.22",
```

Comment:
It doesn't _really_ matter for applications, but other `@types` packages are listed in devDependencies.

Reply:
oh that's interesting.  I'll start doing that in the tdr UI repo too!